airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/spec.json

```diff
@@ -0,0 +1,46 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/zendesk-support",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Source Zendesk Support Spec",
+    "type": "object",
+    "required": ["start_date", "subdomain", "auth_method"],
```

Comment:
I think the start date should not be required. You can place the "default" field with a value like "1970-01-01T00:00:00Z" and it will be the start point from where data will replicate.

Reply:
this is the singer current settings. We has to migrate its functionality as is.